src/openforms/appointments/tokens.py

```diff
@@ -0,0 +1,19 @@
+from django.utils import timezone
+
+from openforms.appointments.models import AppointmentInfo
+from openforms.submissions.tokens import SubmissionTokenGenerator
+
+
+class SubmissionAppointmentTokenGenerator(SubmissionTokenGenerator):
```

Comment:
Let's implement this entire generator here rather than subclassing - there's too much stuff in motion at the moment that will cause confusion.